ledger/src/shred.rs

```diff
@@ -337,6 +337,8 @@ impl Shred {
     dispatch!(pub fn payload(&self) -> &Vec<u8>);
     dispatch!(pub fn sanitize(&self) -> Result<(), Error>);
 
+    dispatch!(pub fn merkle_root(&self) -> Option<Hash>);
```

Comment:
Can you please keep lines 336-340 alphabetically sorted?
also, the blank like 339 is not needed.

also, the return type should be `Result<Hash, Error>` to preserve the error.